modules/admin_manual/pages/installation/server_prep_ubuntu_18.04.adoc

```diff
@@ -0,0 +1,199 @@
+= Server Preparation for Ubuntu 18.04
+:hash-installation: http://php.net/manual/en/hash.installation.php
+:mcrypt-link-url: https://websiteforstudents.com/install-php-7-2-mcrypt-module-on-ubuntu-18-04-lts/
+:mcrypt-pecl-url: https://pecl.php.net/package/mcrypt
+:discover-samba-hosts: https://ubuntuforums.org/showthread.php?t=2384959
+:install-mariadb-latest: https://downloads.mariadb.org/mariadb/repositories/#
+
+== Webserver and php
+
+You only need to install __one__ of the following webservers.
+ 
+=== Apache Webserver + PHP (supported)
+
+The following command installs Apache webserver and php.
+
+[source,console]
+----
+sudo apt install php libapache2-mod-php apache2
+----
+
+=== Apache Webserver + PHP-FPM (unsupported)
+
+The following command installs and enables Apache webserver with php-fpm. +
+This setup can be used in high thruput environments. For performance
+comparison details please see following
+https://www.cloudways.com/blog/php-fpm-on-cloud/[link].
+
+[source,console]
+----
+sudo apt install php-fpm apache2
+sudo a2enmod proxy_fcgi
+sudo a2enconf php7.2-fpm
+sudo a2dismod php7.2
+sudo a2dismod mpm_prefork
+sudo a2enmod mpm_event
+sudo a2enmod http2
+sudo service apache2 restart
+----
+
+For examples how to configure this setup see 
+http://httpd.apache.org/docs/2.4/mod/mod_proxy.html#handler[link 1] and
+https://httpd.apache.org/docs/2.4/mod/mod_proxy_fcgi.html[link 2] and
+https://wiki.apache.org/httpd/PHP-FPM[link 3]
+
+=== Nginx Webserver + PHP-FPM (unsupported)
+
+The following command installs nginx webserver and php-fpm.
+
+[source,console]
+----
+sudo apt install php-fpm nginx
+----
+
+For details how to configure nginx please see the
```

Comment:
Please change this sentence to: "For details of how to configure Nginx, please see the"